### PR TITLE
Upgrade application from LB2 to LB3

### PR DESCRIPTION
**Pull request by jisoolee**
Update for loopback version 3

### CODE REVIEW

**Comment by sam-github** [comment]
`fix(dep): ...` suggests a patch release to me, but this bumps the package major. Just a note, I'm not involved in maintaining this stuff.

**Review by bajtos — APPROVED** [review]
I am not familiar with this codebase. I don't see any obvious problems and the tests pass, so LGTM.

Kudos for bumping-up the major version :+1:

File: package.json [hunk 1]
```diff
@@ -12,24 +12,26 @@
...
+    "loopback": "^3.x",
     "loopback-boot": "^2.6.5",
-    "loopback-datasource-juggler": "^2.19.0",
-    "serve-favicon": "^2.0.1"
+    "serve-favicon": "^2.0.1",
+    "strong-error-handler": "^1.x"
   },
   "devDependencies": {
-    "eslint": "^1.x",
+    "eslint": "^5.16.0",
```
Comment (rmg) [inline package.json:21]: Is it necessary to update `eslint`?

Comment (b-admike) [reply package.json:21]: For posterity, as discussed, this is because of security vulnerabilities reported by `npm audit`.

**Pull request merged**